mage_ai/data_preparation/models/constants.py

```diff
@@ -18,6 +18,7 @@
 LOGS_DIR = '.logs'
 MAX_PRINT_OUTPUT_LINES = int(os.getenv('MAX_PRINT_OUTPUT_LINES', 1000) or 1000)
 PIPELINE_CONFIG_FILE = 'metadata.yaml'
+PIPELINE_MAX_SIZE = 500000  # maximum size of a pipeline import zip in kb (500Mb)
```

Comment:
This constant is used to test against zip bombs. In the event that the extracted contents of the zip exceed this value, the import process stops. Currently, this limit has been established based on thorough consideration and analysis, but I would appreciate some feedback on this. Thanks.